libs/estdlib/src/calendar.erl

```diff
@@ -113,9 +113,10 @@ date_to_gregorian_days(Year, M, D) when
     Era * 146097 + DoE + 60.
 
 %%-----------------------------------------------------------------------------
-%% @param   DateTime
-%% @returns Days    number of days
-%% @doc     Computes the number of gregorian days starting with year 0 and ending at the specified date.
+%% @param   DateTime the date and time to retrieve seconds
```

Comment:
To me retrieve sounds like extracting a part, as used below for `day_of_the_week/1`, i.e. would yield 0..59 (or 0..60 with leap seconds)

```suggestion
%% @param DateTime the date and time to convert to seconds
```